### PR TITLE
Add meta descriptions to Lisk-SDK, closes #371

### LINKED ISSUE #371: Add meta tags
Add meta tags for every docs page

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #371

### CODE REVIEW

File: modules/ROOT/pages/lisk-commander/index.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 = Lisk Commander Documentation Overview
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Commander overview gives an introduction to the Lisk Commander CLI tool its setup and usage.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-commander/index.adoc]:
```suggestion
:description: The Lisk Commander overview provides an introduction to the Lisk Commander CLI tool its setup and usage.
```

File: modules/ROOT/pages/lisk-commander/user-guide.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = Lisk Commander User Guide
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Commander user guide describes how to use and how to configure the Lisk Commander to suit your needs.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-commander/user-guide.adoc]:
```suggestion
:description: The Lisk Commander user guide describes how to use and configure the Lisk Commander to suit your requirements.
```

File: modules/ROOT/pages/lisk-commander/user-guide/commands.adoc [hunk 3]
```diff
@@ -1,5 +1,6 @@
 = Lisk Commander Reference
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Commander Command Reference lists all available commands of Lisk Commander as a technical reference. For each it comprises all available options, a usage example and an example response.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-commander/user-guide/commands.adoc]:
```suggestion
:description: The Lisk Commander Command Reference lists each available command of Lisk Commander as a technical reference. For all commands this consists of all available options, a usage example and an example response.
```

File: modules/ROOT/pages/lisk-commander/user-guide/lisk-core.adoc [hunk 4]
```diff
@@ -1,5 +1,6 @@
 = Managing Lisk Core
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference about all available commands to manage Lisk Core with Lisk Commander.  For command it lists all available options, a usage example and an example response.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-commander/user-guide/lisk-core.adoc]:
```suggestion
:description: Technical reference regarding each available command used to manage Lisk Core with Lisk Commander.  For all commands all available options, a usage example and an example response are listed.
```

File: modules/ROOT/pages/lisk-commander/user-guide/sensitive-inputs.adoc [hunk 5]
```diff
@@ -1,5 +1,6 @@
 = Lisk Commander Sensitive Inputs
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The sensitive Inputs page describes the different ways how to provide sensitive data like passphrases to Lisk Commander.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-commander/user-guide/sensitive-inputs.adoc]:
```suggestion
:description: The sensitive Inputs page describes the different methods of how to provide sensitive data such as passphrases to the Lisk Commander.
```

File: modules/ROOT/pages/lisk-elements/index.adoc [hunk 6]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Overview
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Elements overview gives an introduction to the Lisk Elements libray and it's different sub-packages. It describes the basic usage and how to upgrade with NPM.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/index.adoc]:
```suggestion
:description: The Lisk Elements overview provides an introduction to the Lisk Elements library and the different sub-packages. The overall basic usage and how to upgrade with NPM is described here..
```

File: modules/ROOT/pages/lisk-elements/packages.adoc [hunk 7]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Packages
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Elements packages page gives an overview about all available packages of Lisk Elements and direct links to their technical reference. It also comprises the basic usage of a Lisk Elements package.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages.adoc]:
```suggestion
:description: The Lisk Elements packages page provides an overview regarding all available packages of Lisk Elements and direct links to their associated technical references. In addition, it also comprises of the basic usage of a Lisk Elements package.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client.adoc [hunk 8]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the API Client package of Lisk Elements. It copmrises usage examples, available options and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client.adoc]:
```suggestion
:description: Technical references regarding the API Client package of Lisk Elements. This consists of usage examples, available options and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/accounts.adoc [hunk 9]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Accounts Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Accounts endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/accounts.adoc]:
```suggestion
:description: Technical references regarding the Account endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/accounts.adoc]:
```suggestion
:description: Technical references regarding the Account endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/accounts.adoc]:
```suggestion
:description: Technical references regarding the Account endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/accounts.adoc]:
```suggestion
:description: Technical references regarding the Account endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/blocks.adoc [hunk 10]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Blocks Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Blocks endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/blocks.adoc]:
```suggestion
:description: Technical references regarding the Blocks endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/dapps.adoc [hunk 11]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: DApps Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the DApps endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/dapps.adoc]:
```suggestion
:description: Technical references regarding the DApps endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/delegates.adoc [hunk 12]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Delegates Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Delegates endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/delegates.adoc]:
```suggestion
:description: Technical references regarding the Delegate endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/node.adoc [hunk 13]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Node Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Node endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/node.adoc]:
```suggestion
:description: Technical references regarding the Nodes endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/peers.adoc [hunk 14]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Peers Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Peers endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/peers.adoc]:
```suggestion
:description: Technical references regarding the Peers endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc [hunk 15]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Signatures Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Signatures endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc]:
```suggestion
:description: Technical references regarding the Signatures endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/transactions.adoc [hunk 16]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Transactions Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Transactions endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/transactions.adoc]:
```suggestion
:description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/voters.adoc [hunk 17]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Voters Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Voters endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/voters.adoc]:
```suggestion
:description: Technical references regarding the Voter endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/votes.adoc [hunk 18]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Votes Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Votes endpoints of the API Client package of Lisk Elements. It copmrises Usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/votes.adoc]:
```suggestion
:description: Technical references regarding the Vote endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/client.adoc [hunk 19]
```diff
@@ -1,5 +1,6 @@
 = Lisk Client
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Client package of Lisk Elements. It copmprises references to all included sub-packages and general usage instructions.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/client.adoc]:
```suggestion
:description: Technical references regarding the Clients packages of Lisk Elements. This consists of references to all included sub-packages and general usage instructions.
```

File: modules/ROOT/pages/lisk-elements/packages/constants.adoc [hunk 20]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Constants
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Constants package of Lisk Elements. It copmrises setup instructions and usage examples.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/constants.adoc]:
```suggestion
:description: Technical references regarding the Constants packages of Lisk Elements. This consists of setup instructions and usage examples.
```

File: modules/ROOT/pages/lisk-elements/packages/cryptography.adoc [hunk 21]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Cryptography
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Cryptography package of Lisk Elements. It copmrises usage examples, available options and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/cryptography.adoc]:
```suggestion
:description: Technical references regarding the Cryptography packages of Lisk Elements. This consists of usage examples, available options and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/passphrase.adoc [hunk 22]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Passphrase
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Pasphrase package of Lisk Elements. It copmrises usage examples, available options and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/passphrase.adoc]:
```suggestion
:description: Technical references regarding the Passphrase package of Lisk Elements. This consists of usage examples, available options and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/transactions.adoc [hunk 23]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements Transactions
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical reference for the Transactions package of Lisk Elements. It copmrises usage examples, available options and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/transactions.adoc]:
```suggestion
:description: Technical references regarding the Transactions packages of Lisk Elements. This consists of usage examples, available options and example responses.
```

File: modules/ROOT/pages/lisk-framework/index.adoc [hunk 24]
```diff
@@ -1,5 +1,6 @@
 = Lisk Framework Overview
 Mona Bärenfänger <mona@lightcurve.io>
+:description: The Lisk Framework overview provides a high-level overview of the Lisk Framework architecture, its modules and components, how they communicate and how to change the default configuration.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-framework/index.adoc]:
```suggestion
:description: The Lisk Framework overview provides a high-level overview of the Lisk Framework architecture, including it's modules and components, how they communicate and how to change the default configuration.
```

**Review by chris529 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc [hunk 25]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Signatures Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical references regarding the Signatures endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc:3]:
```suggestion
:description: Technical references regarding the Signature endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

File: modules/ROOT/pages/lisk-elements/packages/api-client/transactions.adoc [hunk 26]
```diff
@@ -1,5 +1,6 @@
 = Lisk Elements API Client: Transactions Resource
 Mona Bärenfänger <mona@lightcurve.io>
+:description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements.This consists of usage examples, available parameters and example responses.
```
Comment (chris529) [inline modules/ROOT/pages/lisk-elements/packages/api-client/transactions.adoc]:
```suggestion
:description: Technical references regarding the Transactions endpoints of the API Client package of Lisk Elements. This consists of usage examples, available parameters and example responses.
```

**Review by chris529 — DISMISSED** [review]
minor updates

File: modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc (hunk 25 above)
Comment (Tschakki) [reply modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc:3]: I would use plural here, because the endpoint is in plural as well

Comment (chris529) [reply modules/ROOT/pages/lisk-elements/packages/api-client/signatures.adoc:3]: Agreed.

**Review by chris529 — APPROVED** [review]

**Pull request merged**